packages/dev/test/provider/RelayClient.test.ts

```diff
@@ -139,7 +141,8 @@ contract('RelayClient', function (accounts) {
     logger = createClientLogger(loggerConfiguration)
     relayClient = new RelayClient({ provider: underlyingProvider, config: gsnConfig })
     await relayClient.init()
-    gasLess = await web3.eth.personal.newAccount('password')
+    gasLessAccount = relayClient.newAccount()
+    gasLess = toChecksumAddress(gasLessAccount.address)
```

Comment:
this is a bit of an issue. in all cases you replace signing with provider-controlled account (major case) with an ephemeral key (edge case)

I suggest this test transfers all ether from account 9 to a single bit, and then the test is done again with actual RPC call to sign transactions.

Reply:
I thought about that solution of emptying an account as well, but Dror suggested adding an account, so I did. I can change it though.